src/app/components/Promotions/index.tsx

```diff
@@ -50,7 +50,7 @@ export const Promotions: React.FC<PromotionsProps> = ({
             className={cardClassName}
             imageClassName={cardImageClassName}
           /> */}
-          <PromotionCard
+          {/* <PromotionCard
```

Comment:
I'd probably just delete all of the commented out promotions (if we know we don't want to bring them back soon).